resources/lib/ytchannels.py

```diff
@@ -39,7 +39,17 @@ def ytchannels_main():
 
 		YOUTUBE_API_KEY = my_addon.getSetting('youtube_api_key')
 
-	from .functions import build_url, delete_database, get_folders, add_folder, remove_folder, get_channels, get_channel_id_from_uploads_id, add_channel, remove_channel, search_channel, search_channel_by_username, get_latest_from_channel, get_playlists
+	from .functions import build_url, delete_database, get_folders, add_folder, remove_folder, get_channels, get_channel_id_from_uploads_id, add_channel, remove_channel, search_channel, search_channel_by_username, get_latest_from_channel, get_playlists, add_sort_db, init_sort, move_up, move_down
+
+	SORT_INIT = my_addon.getSetting('sort_init')
```

Comment:
If we have a blank DB due to a fresh install, this will run. I think we're probably better off testing for the existance of the column in the DB. If the column is there, we can assume the data is there. If the column isn't then we need to run this migration system.

I think (assuming the bundled version of sqlite is recent enough), the easiest way to do that is: 
```
SELECT 1 FROM pragma_table_info('Channels') WHERE name = 'sort'
```

If that doesn't work, then we might have to use the `PRAGMA table_info(Channels);` query and filter the result in code.

Reply:
New commit should fix this, let me know what you think

Reply:
Cool